Bash Scripts/createCollectionsForAllMembersNested.sh

```diff
@@ -20,34 +20,75 @@ else
 fi
 
 # Set up CLI and API auth
+org_client_secret_key=$(cat secureString_secret.txt \
+	| openssl enc -aes-256-cbc -md sha512 -a -d -pbkdf2 -iter 100000 \
```

Comment:
Could you also increase this to 600,001 iterations?

Reply:
All iterations upgraded to 600001.